web/src/main/java/com/futurewei/alcor/web/entity/dataplane/NetworkConfiguration.java

```diff
@@ -27,16 +27,15 @@
 
   private ResourceType rsType;
   private OperationType opType;
-  boolean allOrNone = true;
 
   @JsonProperty("ports_internal")
   private List<InternalPortEntity> portEntities;
```

Comment:
If you want to rename the variables of vpc and subnet, also rename portEntities to ports.

Reply:
i don't need to rename

Reply:
@haboy52581 Not sure if we are on the same page here. 

As the variable names of vpc, subnets and securitygroups don't come with entity, we should rename portEntities to ports.

Reply:
Create a tracking issue https://github.com/futurewei-cloud/alcor/issues/289. This can be done in a follow-up PR together for L3 routing.